screens/HomeTour/Carousel/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Fragment, useState, useRef } from 'react'
+import { ScrollView, View } from 'react-native'
+import { Dot } from './styled'
+import Slide from './slide'
+
+const Carousel = ({ slides }) => {
+    const slider = useRef(null)
+    const [slidesNumber, setSlideNumber] = useState(1)
+
+
+    return (
+        <Fragment>
+            <ScrollView
+                ref={slider}
+                horizontal
+                pagingEnabled
+                showsHorizontalScrollIndicator={false}
+                onScrollEndDrag={(e) => console.log(e.nativeEvent)}
+            >
+                {slides.map((item) => <Slide text={item.text} image={item.file} />)}
+            </ScrollView>
+            <View style={{ flexDirection: 'row' }}>
+                <Dot style={{ backgroundColor: '#83878d' }} />
```

Comment:
It looks like ... static 😅

Reply:
static ?

Reply:
Always the first dot selected

Reply:
Ouai les puces c est work in progress encore faut que j écoute le Scroll pour faire un truc propre.